Assets/HoloToolkit-Examples/Input/Scenes/Cursor.unity

```diff
@@ -390,7 +388,7 @@ Prefab:
         type: 2}
       propertyPath: Cursor
       value: 
-      objectReference: {fileID: 224284989}
+      objectReference: {fileID: 0}
```

Comment:
This reference should be replaced with the new cursor on Managers -> InputManager's SimpleSinglePointerSelector script.